### ISSUE TITLE
Crashing on JLFork-1.1.7 1.20.1

### ISSUE THREAD

**Issue by SHXRKIEgg** [Senior-S/JustLeveling-Fork#28]
Random Crash occurred. It mentions that is Just Leveling causing it but also mentions lootbeams.

[crash-2024-11-13_00.02.42-server.txt](https://github.com/user-attachments/files/17719252/crash-2024-11-13_00.02.42-server.txt)

**Comment by Senior-S** [comment]
Do you mind testing if removing Otherworld Origins fix this issue?

**Comment by muon-rw** [comment]
I'm not modifying the `get` candidates in `AptitudeCapability` at all 

I *think* this is just because `Minecraft.player` is `@Nullable` and you're not checking for such before you call `getCapability()` 
In [AptitudeCapability](https://github.com/Senior-S/JustLeveling-Fork/blob/41c937d1f91a2cef1cc270081d6f6b02f77b3ad8/src/main/java/com/seniors/justlevelingfork/common/capability/AptitudeCapability.java#L103C1-L105C6):
```
    public static AptitudeCapability get() {
        return (Minecraft.getInstance()).player.getCapability(RegistryCapabilities.APTITUDE).orElseThrow(() -> new IllegalArgumentException("Player does not have Capabilities!"));
    }
```

should probably be something like: 
```
    @Nullable
    public static AptitudeCapability get() {
        LocalPlayer player = Minecraft.getInstance().player;
        if (player == null ) {
            return null;
        }
        return player.getCapability(RegistryCapabilities.APTITUDE).orElseThrow(() -> new IllegalArgumentException("Player does not have Capabilities!"));
    }
```

**Comment by PancakesBatch** [comment]
Disable Sinytra Connector.
It causes the nullable instance.

**Comment by Senior-S** [comment]
Fixed on 1.1.8.
https://modrinth.com/mod/justleveling-fork/versions
Still under review in CurseForge.
